src/hooks/xcm/useXcmBridge.ts

```diff
@@ -125,14 +126,37 @@ export function useXcmBridge(selectedToken: Ref<ChainAsset>) {
     existentialDeposit.value = result;
   };
 
+  const setRelaychainBal = async (): Promise<void> => {
+    if (!relayChainApi || !selectedToken.value) return;
+    await relayChainApi.isReady();
+    const rawBalance = await getRelayChainNativeBal();
+    const decimals = Number(String(selectedToken.value.metadata.decimals));
+    const balance = ethers.utils.formatUnits(rawBalance, decimals).toString();
+    relaychainBal.value = Number(balance);
+  };
+
+  const checkIsEnoughEd = (amount: number): boolean => {
+    if (!existentialDeposit.value?.relaychainMinBal) return false;
+
+    if (isDeposit.value) {
+      const relayBalAfterTransfer = relaychainBal.value - amount;
+      const result = relayBalAfterTransfer > existentialDeposit.value.relaychainMinBal;
```

Comment:
maybe we can refactor this?
```
const relaychainMinBal = existentialDeposit.value?.relaychainMinBal;
```
and instead of use of `result`, just return like
```
return relayBalAfterTransfer > relaychainMinBal;
```

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/436/commits/0d6f2a0036f4b79a0d224d1f6d240ff30d76eddf) commit.